ui/app/AppLayouts/Profile/views/AppearanceView.qml

```diff
@@ -23,12 +23,6 @@ SettingsContentBase {
 
     property var systemPalette
 
-    enum Theme {
-        Light,
-        Dark,
-        System
-    }
```

Comment:
Just curious, why was this removed? Doesn't look like there are any other changes there `Theme` needed to be removed. So I guess it was... just not used?

Reply:
Yeah, not used at all; the current theme is checked like this:
`control.checked: localAppSettings.theme === Universal.System`